CI build fails for net9.0 target: missing apphost binary

### Describe the bug

The CI pipeline fails when building the project for the `net9.0` target framework with the following error:

```
error MSB3030: Could not copy the file ".../obj/Release/net9.0/apphost" because it was not found.
```

The project multi-targets both `net10.0` and `net9.0` (defined in `Directory.Build.props`), but the CI workflow only installs the .NET 10 SDK via `global.json`. The .NET 10 SDK can compile code targeting `net9.0`, but it does not include the native `apphost` binary required for executable projects (tests, AppHost).

### Expected behavior

The CI build should succeed for all target frameworks.

### Actual behavior

The build fails with `MSB3030` error when trying to copy the `apphost` binary for `net9.0`.

### How to Reproduce?

1. Open a Dependabot PR that modifies `Directory.Packages.props`
2. The CI pipeline triggers on `pull_request` to `main`
3. The build step fails for the `net9.0` target

### Additional information

.NET 9 is an STS (Standard Term Support) release reaching end of support in May 2026. Since the project now targets .NET 10 (LTS) as the default framework, dropping `net9.0` from `TargetFrameworks` is the proposed fix.

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -61,9 +61,6 @@ jobs:
         run: |
           curl -Lo $RUNNER_TEMP/sbom-tool https://github.com/microsoft/sbom-tool/releases/latest/download/sbom-tool-linux-x64
           chmod +x $RUNNER_TEMP/sbom-tool
-          # Generate SBOM for .NET 9.0 build output
-          $RUNNER_TEMP/sbom-tool generate -b ./src/Microcks.Aspire/obj/Release/net9.0 -bc . -pn Microcks.Aspire -pv ${{ github.event.inputs.version }} -ps Microcks -nsb https://microcks.io
-          cp ./src/Microcks.Aspire/obj/Release/net9.0/_manifest/spdx_2.2/manifest.spdx.json ./Microcks.Aspire-net9.0.spdx-sbom.json
           # Generate SBOM for .NET 10.0 build output
           $RUNNER_TEMP/sbom-tool generate -b ./src/Microcks.Aspire/obj/Release/net10.0 -bc . -pn Microcks.Aspire -pv ${{ github.event.inputs.version }} -ps Microcks -nsb https://microcks.io
           cp ./src/Microcks.Aspire/obj/Release/net10.0/_manifest/spdx_2.2/manifest.spdx.json ./Microcks.Aspire-net10.0.spdx-sbom.json

--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -6,7 +6,7 @@
 		<VersionSuffix>alpha</VersionSuffix>
 
 		<DefaultTargetFramework>net10.0</DefaultTargetFramework>
-		<TargetFrameworks>$(DefaultTargetFramework);net9.0</TargetFrameworks>
+		<TargetFrameworks>$(DefaultTargetFramework)</TargetFrameworks>
 
 		<Description>Microcks.Aspire for .NET</Description>
 		<Authors>Sébastien DEGODEZ</Authors>

